jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-backend/src/test/java/org/jbpm/workbench/cm/backend/server/RemoteCaseManagementServiceImplTest.java

```diff
@@ -326,4 +331,78 @@ private CaseMilestone createTestMilestone(String caseMilestoneId, String caseMil
 
         return milestone;
     }
+
+    private TaskSummary createTaskSummary(Long taskId, String name, String status) {
+        TaskSummary taskSummary = TaskSummary.builder()
+                .id(taskId)
+                .name(name)
+                .status(status)
+                .createdOn(new Date())
+                .build();
+
+        return taskSummary;
+    }
+
+    private CaseAdHocFragment createAdHocFragment(String name, String type) {
+        CaseAdHocFragment caseAdHocFragment = CaseAdHocFragment.builder()
+                .name(name)
+                .type(type)
+                .build();
+
+        return caseAdHocFragment;
+    }
+
+    private void assertActionEquals(TaskSummary taskSummary, CaseActionSummary caseActionSummary) {
+        assertNotNull(taskSummary);
+        assertNotNull(caseActionSummary);
+        assertEquals(taskSummary.getId(), caseActionSummary.getId());
+        assertEquals(taskSummary.getName(), caseActionSummary.getName());
+        assertEquals(taskSummary.getStatus(), caseActionSummary.getStatus());
+    }
+
+    private void assertActionEqualsAdHocFragment(CaseAdHocFragment caseAdHocFragment, CaseActionSummary caseActionSummary) {
+        assertNotNull(caseAdHocFragment);
+        assertNotNull(caseActionSummary);
+        assertEquals(caseAdHocFragment.getType(), caseActionSummary.getDescription());
+        assertEquals(caseAdHocFragment.getName(), caseActionSummary.getName());
+        assertEquals("AD_HOC", caseActionSummary.getStatus());
+    }
+
+    @Test
+    public void getCaseActionsTest() {
+        String userId = "userId";
```

Comment:
Maybe replacing this local variable with a class field with the same name, so it's aligned with other variables used in the test?